libs/api/__tests__/specs/mailchimpMailProvider.test.ts

```diff
@@ -1,32 +1,60 @@
-import {MailchimpMailProvider} from '../../src'
-
-const mockSend = jest.fn().mockImplementationOnce(cb => cb())
-const mockSendTemplate = jest.fn().mockImplementationOnce(cb => cb())
-jest.mock('mandrill-api', () => {
-  return {
-    Mandrill: jest.fn().mockImplementation(() => {
-      return {
-        messages: {
-          send: mockSend,
-          sendTemplate: mockSendTemplate
-        }
-      }
-    })
-  }
-})
-
-jest.mock('../../src/lib/server', () => {
-  const originalModule = jest.requireActual('../../src/lib/server')
-  return {
-    __esModule: true,
-    ...originalModule,
-    logger: jest.fn(() => ({error: jest.fn()}))
-  }
-})
+import nock from 'nock'
```

Comment:
We already have supertest, could we use that instead or does supertest not work?